maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -140,8 +146,29 @@ object MaestroSessionManager {
                         val channel = ManagedChannelBuilder.forAddress(defaultHost, idbPort)
                             .usePlaintext()
                             .build()
+
+                        val xcTestDriverClient = XCTestDriverClient(defaultHost, xcTestPort)
```

Comment:
Can we test with the firewall that this is not giving the permission dialog for external network communication?

Reply:
This change does not affect anything fundamentally as XCUITest server is still started on `.loopback` address. This is only needed for Cloud where we will do port forwarding